packages/bot-web-ui/src/components/run-panel.jsx

```diff
@@ -176,14 +175,40 @@ class RunPanel extends React.PureComponent {
                     {content}
                 </Drawer>
                 {is_mobile && <MobileDrawerFooter {...this.props} should_show_statistics={active_index !== 2} />}
-                <Dialog
+                <Modal
                     title={dialog_options.title}
                     is_open={is_dialog_open}
-                    onOkButtonClick={onOkButtonClick}
-                    onCancelButtonClick={onCancelButtonClick}
+                    toggleModal={onCancelButtonClick || onOkButtonClick}
+                    className='no-border'
+                    has_close_icon
+                    width={is_mobile ? 'calc(100vw - 48px)' : '440px'}
                 >
-                    {dialog_options.message}
-                </Dialog>
+                    <Modal.Body>{dialog_options.message}</Modal.Body>
+                    <Modal.Footer>
+                        {onCancelButtonClick && (
+                            <Button
+                                large
+                                text={
+                                    dialog_options.cancel_button_text
+                                        ? dialog_options.cancel_button_text
+                                        : localize('Cancel')
+                                }
+                                onClick={() => {
+                                    onCancelButtonClick();
+                                }}
+                                secondary
+                            />
+                        )}
+                        <Button
```

Comment:
No it's not necessary, I used onCancelButtonClick because in run-panel store we have dialogs(e.g showRealAccountDialog ) which have  onCancelButtonClick = undefined bot all dialogs have onOkbuttonClick